packages/runtime/container-runtime/src/summarizerTypes.ts

```diff
@@ -221,3 +232,56 @@ export interface ISummarizer
         options: Omit<ISubmitSummaryOptions, "summaryLogger">,
     ): OnDemandSummarizeResult;
 }
+
+/** Data about an attempt to summarize. */
+export interface ISummaryAttempt {
```

Comment:
nit: Consider renaming to `ISummarizeAttempt` to keep it consistent with other names.

Reply:
Okay I renamed it.